Exception tracking: no reported exceptions in the past week (#39)
madetech/productionisation: PRODUCTIONISATION.md

* [ ] 7.2 Alerts sent to relevant instant chat channel
* [ ] 7.3 Alerts sent to relevant team email address
* [ ] 7.4 UUID of request is sent as additional user data

> * [ ] 7.5 No reported unhandled exceptions in the past week

Reviewer: This is ambiguous.

Are we saying:
- No exceptions in Sentry that haven't been addressed within a week
- No exceptions in Sentry for at least a week

Author: Updated with alternative wording, and added additional justification for the point.

* [ ] 7.2 Alerts sent to relevant instant chat channel
* [ ] 7.3 Alerts sent to relevant team email address
* [ ] 7.4 UUID of request is sent as additional user data

> * [ ] 7.5 No unresolved exceptions in the past week  

Reviewer: ```suggestion
* [ ] 7.5 reported exceptions are resolved within a week
```
This orientates the item to be more around a process + discipline, which is more inline with what the description is talking about.

>   _Exceptions that have been thrown but not caught or handled safely could lead to larger problems for applications and teams. It's good for teams to have the discipline of regularly checking, or acting upon reported exceptions as they occur during development to prevent these from accruing. If code changes have been made to handle these, the exceptions should be marked as resolved in the tracking service. No reported exceptions in the past week is a positive sign of well managed exceptions._

Reviewer: The item is for "no unresolved exceptions", however the description goes on to talk about "no reported exceptions".

having "no exceptions" is an unreasonable thing to request.
